share/config.fish

```diff
@@ -152,3 +152,15 @@ function . --description 'Evaluate contents of file (deprecated, see "source")'
 		source $argv
 	end
 end
+
+# As last part of initialization, source the conf directories
+# Implement precedence (User > Admin > Vendors > Fish) by basically doing "basename"
+set -l sourcelist
+for file in $configdir/fish/conf.d/* $__fish_sysconfdir/conf.d/* $__fish_datadir/vendor_conf.d/*
```

Comment:
@jrobeson: This actually works in that case because the glob currently expands to nothing (try `echo /arglbargl/*` in a script - interactively it'll warn, but in a script it'll work). With #2719 as solution to #2394 it will continue to work. Yes, if we go with something else it might need to be revisited.